gsa/src/gmp/models/__tests__/user.js

```diff
@@ -118,10 +118,10 @@ describe('User model tests', () => {
     const user2 = new User(elem2);
     const user3 = new User({});
 
-    expect(user1.auth_method).toEqual(AUTH_METHOD_LDAP);
-    expect(user2.auth_method).toEqual(AUTH_METHOD_RADIUS);
+    expect(user1.authMethod).toEqual(AUTH_METHOD_LDAP);
+    expect(user2.authMethod).toEqual(AUTH_METHOD_RADIUS);
     expect(user1.sources).toBeUndefined();
-    expect(user3.auth_method).toEqual(AUTH_METHOD_PASSWORD);
+    expect(user3.authMethod).toEqual(AUTH_METHOD_PASSWORD);
```

Comment:
We could even add a check for having converted the auth_method property to the tests :-)

```suggestion
    expect(user3.auth_method).toBeUndefined()
```

Reply:
Looking at model data from the tests, there doesn't appear to be an original auth_method. we determined the authMethod based on a property called "sources". Perhaps something like this? `expect(user3.sources).toBeUndefined();`

Reply:
Ah ok. It isn't parsed from an auth_method property. So we don't need another test for this :-)